components/raftstore/src/store/metrics.rs

```diff
@@ -778,4 +778,10 @@ lazy_static! {
         "Sum of applying sst.",
         &["type"]
     ).unwrap();
+
+    pub static ref SNAPSHOT_LIMIT_GENERATE_BYTES: IntCounter = register_int_counter!(
+        "tikv_snapshot_limit_generate_bytes",
```

Comment:
where is the limit?

Reply:
It is the io limit  to limit the snapshot disk. https://github.com/tikv/tikv/blob/da5396b2c2e3ae985cf195f35f52d955d45cd381/components/raftstore/src/store/snap/io.rs#L111-L118